### PR TITLE
[PR on instrumentation_v3] Convert all subparameters into Parameter

### PR DESCRIPTION
**Pull request by jrapin**
Convert all non-parameters to constants

### CODE REVIEW

**Comment by jrapin** [comment]
this removes 8 `isisntance`

**Comment by jrapin** [comment]
@danthe3rd do you prefer it this way?

File: nevergrad/parametrization/core.py [hunk 1]
```diff
@@ -444,10 +447,10 @@ def value(self, value: t.Dict[str, t.Any]) -> None:
             as_parameter(self._parameters[key]).value = val
 
     def get_value_hash(self) -> t.Hashable:
-        return tuple(sorted((x, y.get_value_hash()) for x, y in self._parameters.items() if isinstance(y, Parameter)))
+        return tuple(sorted((x, y.get_value_hash()) for x, y in self._parameters.items() if not isinstance(y, Constant)))
```
Comment (danthe3rd) [inline nevergrad/parametrization/core.py]: I understand you might not need to hash the value of constants, but maybe the user needs? If he has 2 Parameters with different constants inside? (PLUS we remove one `isinstance` call :D)

Comment (jrapin) [reply nevergrad/parametrization/core.py]: Could be useful indeed, however that won't remove an isinstance :p We can only hash what is hashable, so we'll need to check it is...

Comment (jrapin) [reply nevergrad/parametrization/core.py]: Not an isinstance actually then, but an additional try except and a placeholder if it's not easily hashable, and that works ;)

**Pull request merged**